.travis.yml

```diff
@@ -5,8 +5,12 @@ services:
 cache:
   directories:
   - .glide
-script:
-- make verify build test images
+jobs:
+  include:
+    - stage: Verify and Build
+      script: make verify build
+    - stage: Test and Make Image
+      script: make test images
```

Comment:
@duglin Thanks for your insight, yes stages are for independent logically grouped jobs to be performed, this feature is in beta stage and going ahead there might be a time when you can run 2 stages in parallel, at that point of time it will save lot of time. This PR is just being a future ready for Travis CI.